R/slide.R

```diff
@@ -145,6 +144,10 @@ epi_slide = function(x, f, ..., n = 7, ref_time_values,
   time_range = range(unique(x$time_value))
   starts = in_range(ref_time_values - before_num, time_range)
   stops = in_range(ref_time_values + after_num, time_range)
+  
+  if(length(starts) == 0 | length(stops) == 0){ 
```

Comment:
A couple things here:
1. Add a space between `if` and `(`. also between `)` and `{`. It just looks better.
2. For `if` conditionals, use `||` rather than `|`. (the same for `&&`). See the Details in the help for `&`.